docs/reference/pf-command-reference.md

```diff
@@ -905,18 +905,22 @@ pf service start [--port]
 ```
 
 #### Examples
-Start prompt flow service. We endeavor to start the service on the default port 23333. And if it's already taken, we will sequentially probe new ports, incrementing by one each time.
+Prompt flow will try to start the service on the default port 23333 if "pfs.port" file doesn't exist. If the port is 
+already taken, prompt flow will sequentially probe new ports, incrementing by one each time. If the port file exists, 
+prompt flow will just try to start the service on the port number specified in the file.
 
 ```bash
 pf service start
 ```
 
-Force restart promptflow service.
+Forcefully restart the promptflow service even if the port is already in use.
```

Comment:
we don't always restart. rephrase it like: 

Forcefully start the promptflow service. If the port is already in use, xxx